src/Decimal.php

```diff
@@ -7,7 +7,12 @@
 
 class Decimal implements JsonSerializable
 {
-    public const MAX_SCALE = 2147483647; // 2^32-1
+    /**
+     * @var int|null
+     */
+    protected static $default_scale;
+
+    public const MAX_SCALE = PHP_INT_MAX;
```

Comment:
It's not used currently, but is it needed?

Reply:
Not sure yet. We dont do cleanup right now.